packages/config/src/constants.js

```diff
@@ -0,0 +1,114 @@
+const SINGLE_CELL_WITH_HEATMAP_VIEWS = {
+  obsSets: [4, 0, 4, 4],
```

Comment:
```suggestion
  obsSets: { x: 4, y: 0, w: 4, h: 4 },
```

We should use objects so that it is clear what the elements of the array mean ( i just put x/y/w/h for the purpose of the suggestion but i am not sure the true intended ordering)